src/wp-includes/class-wp-theme-json.php

```diff
@@ -2735,6 +2735,8 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 	 * @since 6.6.0 Setting a min-height of HTML when root styles have a background gradient or image.
 	 *              Updated general global styles specificity to 0-1-0.
 	 *              Fixed custom CSS output in block style variations.
+	 * @since 6.6.1 Avoid applying `:root :where()` wrapper to top-level element-only selectors.
+	 * @since 6.6.2 Avoid applying `:root :where()` wrapper to root selectors.
```

Comment:
These 2 `@since` shouldn't be necessary, as they are both bugfixes. `@since` is added for changes in functionality,  function parameter(s), return types, filters, etc.

The commit history will provide the context of the bugfix changes.

```suggestion
```